tag.go

```diff
@@ -101,8 +103,11 @@ func parseTag(fieldName string, structTag reflect.StructTag) (p *tagProperty, is
 	p.dft = tag.Get(tagDefaut)
 
 	// `ignoreDft` TAG
-	toIgnore, _ := strconv.ParseBool(tag.Get(tagDefautInject))
-	p.ignoreDft = toIgnore
+	dftToIgnore, _ := strconv.ParseBool(tag.Get(tagIgnoreDefault))
+	p.ignoreDft = dftToIgnore
+
+	requiredToIgnore, _ := strconv.ParseBool(tag.Get(tagIgnoreRequired))
```

Comment:
where is the error checking?

Reply:
if someone puts something other than "1", "t", "T", "true", "TRUE", "True", "0", "f", "F", "false", "FALSE", "False" for example "" then we'll assume it's false

Reply:
consider typo in the tag - should return error

Reply:
done